packages/types/src/global.ts

```diff
@@ -10,6 +10,7 @@ type ApiAt = ApiDecoration<'promise'> & {rpc: ApiPromise['rpc']};
 
 declare global {
   const api: ApiAt;
+  const unsafeApi: ApiPromise;
```

Comment:
I think this would be 
```suggestion
  const unsafeApi: ApiPromise | undefined;
```